src/org/thoughtcrime/securesms/LogViewActivity.java

```diff
@@ -32,7 +32,7 @@ protected void onCreate(Bundle savedInstanceState) {
     dynamicLanguage.onCreate(this);
 
     setContentView(R.layout.log_view_activity);
-    logViewFragment = LogViewFragment.newInstance();
+    logViewFragment = new LogViewFragment(dynamicLanguage);
```

Comment:
`LogViewFragment.newInstance()` did nothing but `new LogViewFragment(dynamicLanguage)` ... cleaned that up on the way.